mysql-test/suite/galera_3nodes/t/galera_duplicate_primary_value.test

```diff
@@ -0,0 +1,84 @@
+--source include/galera_cluster.inc
+--source include/have_debug.inc
+--source include/have_debug_sync.inc
+--source include/big_test.inc
+--source include/force_restart.inc
+
```

Comment:
why force restart here? test suite execution will be slower by this

Reply:
its not really needed so I will remove it